### PR TITLE
Review suggestions for PR-514 (Custom roles and permissions)

### PR DESCRIPTION
**Pull request by alexisintech**
[PR-514](https://github.com/clerk/clerk-docs/pull/514) was merged when V4 was merged, but still needed a review on its content. 

decisions include:
- roles and permissions are both lowercase, not capitalized (to follow suit of our Dashboard)
- role names, such as Creator, Admin, Member, are bolded
- <Protect /> should be written as such, not as <Protect> (see styleguide rule 4.7)
- removal of "e.g." in favor of "such as" or "for example" (as they are latin terms and may not be understood internationally)

**Lint checker will fail** until [this PR](https://github.com/clerk/clerk-docs/pull/565) is merged (this PR introduces remark-frontmatter so that the mdx linter/link checker ignores frontmatter).

@rachelnabors to follow up on these action items:
[X] [incorrectly labeled image](https://github.com/clerk/clerk-docs/blob/7cf3e15c267f41258738c8d53f99fbc684fd83dd/docs/organizations/creator-role.mdx#L43)
[X] [update an example to include the newly added properties](https://github.com/clerk/clerk-docs/blob/7cf3e15c267f41258738c8d53f99fbc684fd83dd/docs/references/nextjs/authentication-object.mdx#L29)

### CODE REVIEW

File: docs/organizations/roles-permissions.mdx [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
-Clerk supports modeling your own custom Roles and Permissions to control access to resources within your application when you use [Organizations](/docs/organizations/overview).
+Clerk supports modeling your own custom role and permissions to control access to resources within your application when you use [organizations](/docs/organizations/overview).
 
 ## Roles
 
-Roles determine a user's level of access to the Organization. You can extend a Roles privileges and access to resources by adding [Permissions](#permissions).
+Roles determine a user's level of access to the organization. You can extend a role's privileges and access to resources by adding [permissions](#permissions).
 
-### Default Roles
+### Default role
```
Comment (rachelnabors) [inline docs/organizations/roles-permissions.mdx:14]:
```suggestion
### Default roles
```

**Review by rachelnabors — CHANGES_REQUESTED** [review]

**Comment by alexisintech** [comment]
Waiting on one last action item!

**Comment by alexisintech** [comment]
Update 12/15/2023: action item is complete and that PR is merged. This PR is now all set to be merged!

**Review by rachelnabors — APPROVED** [review]
Adding image. Another PR solves the remaining task. Please merge!

**Pull request merged**